README.md

```diff
@@ -434,6 +434,14 @@ docker build \
   ./
 ```
 
+### Log level
+
+To manage logging level user environment variables ``LOG_LEVEL`` possible values are
```

Comment:
"Use the `LOG_LEVEL` environment variable to control the logging verbosity. Possible values are:"